ui/ts/models/status.ts

```diff
@@ -150,6 +160,16 @@ module Models {
         return Object.keys(this._data.result()).sort();
       }
 
+      public GetAllStatuses(): Proto.NodeStatus[] {
```

Comment:
the node/store thing here looks very similar but the code is duplicated. Can some code be shared?